05.ls/ls_command.rb

```diff
@@ -47,21 +46,21 @@ def make_l_option_file(_path)
 end
 
 def show_l_option(file_data)
-  max_nlink_size = file_data.max_by { |file_datum| file_datum[:nlink].size }[:nlink].size - 5
-  max_username_size = file_data.max_by { |file_datum| file_datum[:user_name].size }[:user_name].size + 1
-  max_groupname_size = file_data.max_by { |file_datum| file_datum[:group_name].size }[:group_name].size + 2
-  max_size_size = file_data.max_by { |file_datum| file_datum[:size].size }[:size].size - 2
-  puts "total #{file_data.sum { |file_datum| file_datum[:blocks] }}"
-  file_data.each do |file_datum|
-    print file_datum[:file_type]
-    print file_datum[:permission].ljust(10)
-    print file_datum[:nlink].to_s.rjust(max_nlink_size)
-    print file_datum[:user_name].rjust(max_username_size)
-    print file_datum[:group_name].rjust(max_groupname_size)
-    print file_datum[:size].to_s.rjust(max_size_size)
-    print " #{file_datum[:date]}"
-    print " #{file_datum[:file_name]}"
-    print file_datum[:link]
+  max_nlink_size = file_data.max_by { |file_data_list| file_data_list[:nlink].size }[:nlink].size
+  max_username_size = file_data.max_by { |file_data_list| file_data_list[:user_name].size }[:user_name].size + 1
+  max_groupname_size = file_data.max_by { |file_data_list| file_data_list[:group_name].size }[:group_name].size + 2
+  max_size_size = file_data.max_by { |file_data_list| file_data_list[:size].size }[:size].size
+  puts "total #{file_data.sum { |file_data_list| file_data_list[:blocks] }}"
+  file_data.each do |file_data_list|
```

Comment:
```suggestion
  file_data_list.each do |file_data|
```

data_listが配列、dataがスカラ値（非配列）なので、こういう命名にしてください〜。